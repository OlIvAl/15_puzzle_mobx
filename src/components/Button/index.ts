import styled from 'styled-components';

const StyledButton = styled.button`
  border-radius: 4px;
  padding: 6px 10px;
  background-color: #0d9095;
  color: #fff;
  text-decoration: none;
  border: none;
  
  &:hover {
    background-color: #17bbc2;
  }
  &:disabled {
    background-color: #999;
  }
`;

export default StyledButton;