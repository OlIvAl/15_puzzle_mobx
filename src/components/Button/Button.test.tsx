import * as React from 'react';
import Button from '.';
import renderer from 'react-test-renderer';

describe('Button', () => {
  it('он отображается', () => {
    const tree = renderer.create(<Button />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});