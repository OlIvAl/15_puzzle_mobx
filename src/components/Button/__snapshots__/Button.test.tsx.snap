// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button он отображается 1`] = `
.c0 {
  border-radius: 4px;
  padding: 6px 10px;
  background-color: #0d9095;
  color: #fff;
  -webkit-text-decoration: none;
  text-decoration: none;
  border: none;
}

.c0:hover {
  background-color: #17bbc2;
}

.c0:disabled {
  background-color: #999;
}

<button
  className="c0"
/>
`;
